### DISCUSSION TITLE
Search node boots into Emergency Mode. Root account is locked.

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#3129] [Unsupported Versions]
![9F8A26E1](https://user-images.githubusercontent.com/33126910/109159366-25262f00-773a-11eb-9223-7cca6344462a.PNG)

![image](https://user-images.githubusercontent.com/33126910/109159713-8d751080-773a-11eb-9bb4-9d1ce0c61b2a.png)

This started happening after a month of using the node. Ran soup yesterday and this server wouldn't come back up.

Any suggestions greatly appreciated!

**Comment by weslambert** [comment]
Did the box happen to be recently updated or any files modified?  You may also want to check your filesystem(s) via fsck/LiveCD.

**Comment by juddcbrown** [comment]
Yes, I added a line to /etc/fstab to mount a Synology drive. Must have
botched it.

Am rebuilding the server now, but I appreciate the response. Thanks Wes.

On Fri, Feb 26, 2021 at 07:11 weslambert <notifications@github.com> wrote:

> Did the box happen to be recently updated or any files modified? You may
> also want to check your filesystem(s) via fsck/LiveCD.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/3129#discussioncomment-409169>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH4XT7QRY6JHYMPCZZY7YE3TA6MZZANCNFSM4YGQDXYA>
> .
>
-- 
Judd Brown

**Reply by TOoSmOotH** [reply]
I would add nofail to the fstab for that device as well so that if it happens again it will still boot.
